overdrive.py

```diff
@@ -1120,4 +1120,11 @@ def process_item(self, identifier):
             e = "Could not extract metadata from Overdrive data: %r" % info
             return self.failure(identifier, e)
 
+        self.metadata_pre_hook(metadata)
         return self.set_metadata(identifier, metadata)
+
+    def metadata_pre_hook(self, metadata):
+        """A hook method that allows subclasses to modify a Metadata
+        object derived from Overdrive before it's applied.
+        """
+        return metadata
```

Comment:
This allows the metadata wrangler's Overdrive coverage provider to strip out the CirculationData (assuming Overdrive provides any), so it doesn't think it should be serving books and create an Overdrive LicensePool.